src/upnp_xml.cc

```diff
@@ -473,6 +477,26 @@ std::string UpnpXMLBuilder::renderExtension(const std::string& contentType, cons
     return "";
 }
 
+std::vector<std::shared_ptr<CdsResource>> UpnpXMLBuilder::getOrderedResources(const std::shared_ptr<CdsItem>& item)
+{
+    // Order resources according to index defined by orderedHandler
+    std::vector<std::shared_ptr<CdsResource>> orderedResources {};
```

Comment:
This is somewhat of a minor point, but there's no need to use {} with vector like this.